samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor

```diff
@@ -357,7 +361,7 @@
         {
             if (node.Id == samples.NodeId)
             {
-                node.Value = $"{samples.Value?.ToString()} ({samples.Status}/{samples.Timestamp})";
```

Comment:
>/{samples.Timestamp [](start = 76, length = 19)

I think showing the stamp is valuable especially for diagnostics - the user has a feedback when the value been last refreshed.

Reply:
Oops - overlooked this also.   Please do not undo code during merge, please merge, or ask person making change about context.

Reply:
Sorry I didn't realize it was removed during the merge